src/components/AdminTherapist/AddPsychologist/AddPsychologist.jsx

```diff
@@ -1,11 +1,246 @@
+				setLoading(false);
+			});
+	};
 	return (
-		<div className={styles.mainContainer}>
-			<div className={styles.contForm}>
-				<h1>Psicologos</h1>
+		<div className={styles.contNewTherapist}>
+			<div className={styles.newTherapist}>
+				<h2>Registro de psicologos</h2>
+				<form ref={forms} className={styles.contForm} onSubmit={handleSubmit(handleAdd)}>
+					<div className={styles.contInput}>
+						<p>Nombre: </p>
+						<div className={styles.inputValid}>
+							<input
+								id="name"
+								name="name"
+								{...register('name', {
+									pattern: {
+										message: 'El formato es incorrecto',
+										value: /^[a-zA-Z ]+$/i,
+									},
+									required: {
+										message: 'El nombre es un campo obligatorio',
+										value: true,
+									},
+								})}
+								onChange={handleChange}
+								type="text"
+								value={form.name}
+							/>
+							{errors.name && (<strong>{errors.name.message}</strong>
+							)}
+						</div>
+					</div>
+					<div className={styles.contInput}>
+						<p>Segundo nombre: </p>
+						<div className={styles.inputValid}>
+							<input
+								id="middleName"
+								onChange={handleChange}
+								type="text"
+								value={form.middleName}
+							/>
+						</div>
+					</div>
+					<div className={styles.contInput}>
+						<p>Apellido: </p>
+						<div className={styles.inputValid}>
+							<input
+								id="lastName"
+								name="lastName"
+								{...register('lastName', {
+									pattern: {
+										message: 'El formato es incorrecto',
+										value: /^[a-zA-Z ]+$/i,
+									},
+									required: {
+										message: 'El apellido es un campo obligatorio',
+										value: true,
+									},
+								})}
+								onChange={handleChange}
+								type="text"
+								value={form.lastName}
+							/>
+							{errors.lastName && <strong>{errors.lastName.message}</strong>}
+						</div>
+					</div>
+					<div className={styles.contInput}>
+						<p>Email: </p>
+						<div className={styles.inputValid}>
+							<input
+								id="email"
+								name="email"
+								{...register('email', {
+									pattern: {
+										message: 'El formato del E-mail es incorrecto',
+										value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i,
+									},
+									required: {
+										message: 'El email es un campo obligatorio',
+										value: true,
+									},
+								})}
+								onChange={handleChange}
+								type="text"
+								value={form.email}
+							/>
+							{errors.email && <strong>{errors.email.message}</strong>}
+						</div>
+					</div>
+					<div className={styles.contInput}>
+						<p>Telefono: </p>
+						<div className={styles.inputValid}>
+							<input
+								id="phone"
+								name="phone"
+								{...register('phone', {
+									pattern: {
+										message: 'El formato es incorrecto',
+										value: /^[0-9]+$/i,
+									},
+									required: {
+										message: 'El telefono es un campo obligatorio',
```

Comment:
El **teléfono** es un campo obligatorio
```suggestion
										message: 'El teléfono es un campo obligatorio',
```